baywatch.install

```diff
@@ -389,3 +389,16 @@ function baywatch_update_8018() {
     }
   }
 }
+
+/**
+ * Corrects excluded files path in shield setting.
+ */
+function baywatch_update_8020() {
+  if (\Drupal::moduleHandler()->moduleExists('shield') === TRUE) {
+    $shield = \Drupal::configFactory()->getEditable('shield.settings');
+    $paths = $shield->get('paths');
```

Comment:
please give a if condition to check if path exists then only do the str_replace

Reply:
Will do.